element_lab/lab.py

```diff
@@ -20,8 +20,18 @@ def activate(schema_name, create_schema=True, create_tables=True):
 
 @schema
 class Lab(dj.Lookup):
+    """Table for storing general lab info.
+
+    Attributes:
+        lab ( varchar(24) ): Abbreviated lab name.
+        lab_name ( varchar(255) ): Full lab name.
+        institution ( varchar(255) ): Name of the affiliation institution.
+        address ( varchar(255) ): Physical lab address.
+        time_zone ( varchar(64) ): 'UTC±X' format for NWB export.
```

Comment:
Reading this context, I realize how vague this is (I wrote it).
```suggestion
        time_zone ( varchar(64) ): If using NWB export, use 'UTC±X' format
```